MainRunner.java

```diff
@@ -0,0 +1,89 @@
+package oopFour;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class MainRunner {
+    static Scanner reader = new Scanner(System.in);
+    public static void main(String[] args) {
+
+        ArrayList<Shape> myShapes = new ArrayList<>();
+
+        Double radius = 0.00;
+        Double length = 0.00;
+        Double width = 0.00;
+        Double sideA = 0.00;
+        Double sideB = 0.00;
+        Double sideC = 0.00;
+        Double height = 0.00;
+
+        do {
+            menu();
+            int shapeAnswer = numberFromMenu();
+            if (shapeAnswer == 1) {
+                int numberOfRepeat = secondMenu();
+                for (int i = 0; i < numberOfRepeat; i++) {
+                    System.out.println("Please Enter the Radius of Circle:");
+                    radius = reader.nextDouble();
+                    myShapes.add(new Circle(radius));
+                }
+                finalResult(myShapes);

```

Comment:
No need to run calculation after each entry from the user. You could let user enter all needed shapes, place them in array and then do calculation after user finishes entering new shapes. 
This way, you would do calculation only once which can be important in performance context if more complex operation is done in the area/circumference methods